cmd/tailscale/main.go

```diff
@@ -153,7 +158,7 @@ type Peer struct {
 }
 
 type BackendState struct {
-	Prefs        *ipn.Prefs
+	Prefs        *ipn.PrefsView
```

Comment:
not a pointer. a View is just a pointer-sized struct wrapper around a pointer so it's used as a value type.